libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/task/ConfigureRouterTask.kt

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.navigation.route.onboard.task
+
+import android.os.AsyncTask
+import com.mapbox.navigation.route.onboard.OnOfflineTilesConfiguredCallback
+import com.mapbox.navigation.route.onboard.model.OfflineError
+import com.mapbox.navigator.Navigator
+
+internal class ConfigureRouterTask(
```

Comment:
This is currently implemented but never used. As mentioned above, `MapboxOnboardRouter` is missing some APIs.